examples/router/README.md

```diff
@@ -44,3 +34,15 @@ Take a look at [`Route`](src/main.rs) for the implementation.
 - Detect sub-path from `--public-url` value passed to Trunk. See: thedodd/trunk#51
 
 [`yew-router`]: https://docs.rs/yew-router/latest/yew_router/
+
+## Running
+
+Run a debug version of this application:
+
+```bash
+trunk serve --open
+```
+
+### Notes
+
+Content generation can take up quite a bit of time in debug builds, if it is too slow you should try running with the `release` profile.
```

Comment:
```suggestion
Content generation can take up quite a bit of time in debug builds. If it is too slow, you should try running with the `release` profile.
```

I just marked this one, but there are a couple other sentences that should also be fixed.